modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -1193,7 +1174,7 @@ Query getStyleQuery(FeatureSource<FeatureType, Feature> source,
      * @param styles
      * @return
      */
-    private SortBy[] getSortByFromLiteStyles(LiteFeatureTypeStyle[] styles) {
+    private SortBy[] getSortByFromLiteStyles(List<LiteFeatureTypeStyle> styles) {
         for (LiteFeatureTypeStyle fts : styles) {
             if (fts.sortBy != null) {
                 return fts.sortBy;
```

Comment:
The javadoc already states:

"Returns the sort-by from the list of feature type styles for a given layer. The code assumes the styles have already been classified and are uniform in sorting clauses".

So no changes needed